### ISSUE TITLE
Fixed

### ISSUE THREAD

**Issue by whatdoido** [Stickymaddness/Procurement#675]
Tried deleting the SSID cookie and then reloading, but nothing

**Comment by whatdoido** [comment]
[07-03-2017 20:23] POEApi.Infrastructure.LogonFailedException: Incorrect session id
   at POEApi.Transport.HttpTransport.Authenticate(String email, SecureString password, Boolean useSessionID)
   at POEApi.Model.POEModel.Authenticate(String email, SecureString password, Boolean offline, Boolean useSessionID)
   at Procurement.ViewModel.LoginWindowViewModel.<>c__DisplayClass23_0.<Login>b__0()
   at System.Threading.Tasks.Task.Execute()
[07-03-2017 20:25] POEApi.Infrastructure.LogonFailedException: Incorrect session id
   at POEApi.Transport.HttpTransport.Authenticate(String email, SecureString password, Boolean useSessionID)
   at POEApi.Model.POEModel.Authenticate(String email, SecureString password, Boolean offline, Boolean useSessionID)
   at Procurement.ViewModel.LoginWindowViewModel.<>c__DisplayClass23_0.<Login>b__0()
   at System.Threading.Tasks.Task.Execute()
[07-03-2017 20:26] POEApi.Infrastructure.LogonFailedException: Incorrect session id
   at POEApi.Transport.HttpTransport.Authenticate(String email, SecureString password, Boolean useSessionID)
   at POEApi.Model.POEModel.Authenticate(String email, SecureString password, Boolean offline, Boolean useSessionID)
   at Procurement.ViewModel.LoginWindowViewModel.<>c__DisplayClass23_0.<Login>b__0()
   at System.Threading.Tasks.Task.Execute()
[07-03-2017 20:27] POEApi.Infrastructure.LogonFailedException: Incorrect session id
   at POEApi.Transport.HttpTransport.Authenticate(String email, SecureString password, Boolean useSessionID)
   at POEApi.Model.POEModel.Authenticate(String email, SecureString password, Boolean offline, Boolean useSessionID)
   at Procurement.ViewModel.LoginWindowViewModel.<>c__DisplayClass23_0.<Login>b__0()
   at System.Threading.Tasks.Task.Execute()
[07-03-2017 20:27] POEApi.Infrastructure.LogonFailedException: Incorrect session id
   at POEApi.Transport.HttpTransport.Authenticate(String email, SecureString password, Boolean useSessionID)
   at POEApi.Model.POEModel.Authenticate(String email, SecureString password, Boolean offline, Boolean useSessionID)
   at Procurement.ViewModel.LoginWindowViewModel.<>c__DisplayClass23_0.<Login>b__0()
   at System.Threading.Tasks.Task.Execute()
[07-03-2017 20:27] POEApi.Infrastructure.LogonFailedException: Incorrect session id
   at POEApi.Transport.HttpTransport.Authenticate(String email, SecureString password, Boolean useSessionID)
   at POEApi.Model.POEModel.Authenticate(String email, SecureString password, Boolean offline, Boolean useSessionID)
   at Procurement.ViewModel.LoginWindowViewModel.<>c__DisplayClass23_0.<Login>b__0()
   at System.Threading.Tasks.Task.Execute()
[07-03-2017 20:27] POEApi.Infrastructure.LogonFailedException: Incorrect session id
   at POEApi.Transport.HttpTransport.Authenticate(String email, SecureString password, Boolean useSessionID)
   at POEApi.Model.POEModel.Authenticate(String email, SecureString password, Boolean offline, Boolean useSessionID)
   at Procurement.ViewModel.LoginWindowViewModel.<>c__DisplayClass23_0.<Login>b__0()
   at System.Threading.Tasks.Task.Execute()
[07-03-2017 20:27] POEApi.Infrastructure.LogonFailedException: Incorrect session id
   at POEApi.Transport.HttpTransport.Authenticate(String email, SecureString password, Boolean useSessionID)
   at POEApi.Model.POEModel.Authenticate(String email, SecureString password, Boolean offline, Boolean useSessionID)
   at Procurement.ViewModel.LoginWindowViewModel.<>c__DisplayClass23_0.<Login>b__0()
   at System.Threading.Tasks.Task.Execute()
[07-03-2017 20:27] POEApi.Infrastructure.LogonFailedException: Incorrect session id
   at POEApi.Transport.HttpTransport.Authenticate(String email, SecureString password, Boolean useSessionID)
   at POEApi.Model.POEModel.Authenticate(String email, SecureString password, Boolean offline, Boolean useSessionID)
   at Procurement.ViewModel.LoginWindowViewModel.<>c__DisplayClass23_0.<Login>b__0()
   at System.Threading.Tasks.Task.Execute()
[07-03-2017 20:33] Error downloading account name, exception details: System.ApplicationException: Unable to deserialize object
   at POEApi.Model.POEModel.GetProperObjectFromTransport[T](Stream networkData)
   at POEApi.Model.POEModel.GetAccountName()
[07-03-2017 20:33] System.Exception: Error downloading account name, details logged to DebugInfo.log. Please open a ticket at https://github.com/Stickymaddness/Procurement/issues and include your DebugInfo.log
   at POEApi.Model.POEModel.GetAccountName()
   at Procurement.ViewModel.LoginWindowViewModel.<>c__DisplayClass23_0.<Login>b__0()
   at System.Threading.Tasks.Task.Execute()

**Comment by Needfacelift** [comment]
I had the same issue and after 2 days of messing around doing this seemed to do the trick for me copy the session id then paste it to a sticky note first then recopy it off the sticky note then paste it into procurement.

**Comment by whatdoido** [comment]
It actually worked! thank you!

**Comment by whatdoido** [comment]
I tried it, same problem:
] Error reading character data for veryuniquename, if you are in offline mode you will need to login and update. If you received this error while logging in, the authenticated session may have expired or bad data has been returned by GGG or a network issue may have occurred - Please try again.
The same for every character.

**Comment by aydjay** [comment]
Logout of the path of exile forums.

Login again and grab the cookie.

Make sure there is no trailing or leading white space.

**Comment by whatdoido** [comment]
the problem is not with the sessionid itself, it actually logs in and starts scanning my stash tabs. After it is done, the characters are next, where I have that issue. I tried logging out again and back in, same issue.

**Comment by whatdoido** [comment]
Procurement 1.11.0 Initialized.
                                                          
Authenticating badumts                                                                    [OK]
Fetching account name...                                                                  [OK]
Loading characters...                                                                     [OK]
Loading Standard Stash Tab 1...                                                           [OK]
Loading Standard Stash Tab 2...                                                           [OK]
Loading Standard Stash Tab 3...                                                           [OK]
Loading Standard Stash Tab 4...                                                           [OK]
Loading Standard Stash Tab 5...                                                           [OK]
Loading Standard Stash Tab 6...                                                           [OK]
Loading Standard Stash Tab 7...                                                           [OK]
Loading Standard Stash Tab 8...                                                           [OK]
Loading Standard Stash Tab 9...                                                           [OK]
Loading veryuniquename's inventory...                                                     

[Error] Error reading character data for veryuniquename, if you are in offline mode you will need to login and update. If you received this error while logging in, the authenticated session may have expired or bad data has been returned by GGG or a network issue may have occurred - Please try again.

**Comment by alin-faur** [comment]
I had the same issues since day 1 of patch 2.6.  I used it even in the day of the patch but i couldn't use it after the patch.  The user session id was good but when it got the the new Legacy character it gave the error. I tried for days and it didn't work. Just as I was writing this message I tried again and it worked...the only thing I did different before this attempt is that I logged in the standard league for the first time this patch and then reloged in Legacy. I don't know what was the issue, but there was a bug/issue for sure!
